src/asyncopenstackclient/client.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, api_name, resources, session=None, api_url=None):
         self.api = API(
             api_root_url=self.api_url,
             headers={'X-Auth-Token': self.session.token},
-            json_encode_body=True
+            json_encode_body=True,
+            timeout=60
```

Comment:
maybe it would be better to add only default value here - not fixed, and allow to set it up in NovaClient and GlanceClient args
?